src/common/components/Observations.js

```diff
@@ -76,4 +76,13 @@ const Observations = ({ observations }) => {
   );
 };
 
+const Observations = ({ observations, additionalData }) => {
```

Comment:
This says additionalData but does not define what additionalData is? Can you tell me what the structure of this additionalData parameter is like and what are we using it for?